aidb/vector_database/tasti.py

```diff
@@ -1,21 +1,36 @@
 import numpy as np
 import pandas as pd
-from numba import njit, prange
 from typing import Optional
 
 from aidb.vector_database.vector_database_config import TastiConfig
 
-@njit(parallel=True)
+def _get_and_update_dists_shared(x, i, min_dists, embeddings):
```

Comment:
I think it's better to move it into function 'get_and_update_dists'.

Reply:
Okay @ttt-77 , moved it.